awscli/clidriver.py

```diff
@@ -229,6 +229,7 @@ def _handle_top_level_args(self, args):
                                            format_string=LOG_FORMAT)
             self.session.set_stream_logger('awscli', logging.DEBUG,
                                            format_string=LOG_FORMAT)
+            LOG.debug("Arguments entered to CLI: %s", repr(sys.argv[1:]))
```

Comment:
I can do that. 

The ``repr()`` was just a mind-lapse. I thought I had to convert the list to a string, but I just can enclose it in parenthesis.